source/index.html.md

```diff
@@ -808,6 +808,42 @@ object][ApplePay-PaymentToken] for more information.
   subsequent-in-series authorization.
 </p>
 
+Additionally, an Apple Pay authorization can be created using raw values from
+a payment token:
+
+<dl class="dl-vertical">
+  <dt>applepay[raw][pan]
+    <span class="type">[0-9]{12,19}</span>
+  </dt>
+  <dd>
+    Primary account number of card to charge.
+  </dd>
+  <dt>applepay[raw][expire_month]
+    <span class="type">[0-9]{2}</span>
+  </dt>
+  <dd>
+    Expiry month of card to charge.
+  </dd>
+  <dt>applepay[raw][expire_year]
+    <span class="type">20[0-9]{2}</span>
+  </dt>
+  <dd>
+    Expiry year of card to charge.
+  </dd>
+  <dt>applepay[raw][cryptogram]
+    <span class="type">[:base64:]{28}</span>
```

Comment:
This is the first time we specify a length requirement on a base64 value in the documentation. I don't know if this way is acceptable :slightly_smiling_face:

Reply:
We do this [several](https://github.com/clearhaus/gateway-api-docs/blame/d7203f8be6bc60d82ff0c536ddba27fb5736a06d/source/index.html.md#L557) [other](https://github.com/clearhaus/gateway-api-docs/blame/d7203f8be6bc60d82ff0c536ddba27fb5736a06d/source/index.html.md#L564) [places](https://github.com/clearhaus/gateway-api-docs/blame/d7203f8be6bc60d82ff0c536ddba27fb5736a06d/source/index.html.md#L626) so I think it's perfectly fine 🙂